src/main/kotlin/no/nav/familie/ef/sak/integration/FamilieIntegrasjonerClient.kt

```diff
@@ -30,7 +30,9 @@ class FamilieIntegrasjonerClient(@Qualifier("azure") restOperations: RestOperati
     }
 
     fun hentMedlemskapsinfo(ident: String): Medlemskapsinfo {
-        return postForEntity<Ressurs<Medlemskapsinfo>>(integrasjonerConfig.medlemskapUri, Ident(ident)).data!!
+        val headers = HttpHeaders()
+        headers.add("Nav-Personident", ident)
```

Comment:
Tenker att det hade vært fint hvis vi kunde lagt til en ny endepunkt her hvor vi sender personident i body i stedet. Ser att integrasjoner likevel sender det samme videre, men tenker att det er fint hvis vi har ett fint grensesnitt mot integrasjoner.

Reply:
Ja! Men det er fortsatt inte helt avklart hva vi egentligen trenger av MEDL och dærfør så tenkte vi att vi bruker den eksisterande endepunkten och lager en ny når det blir mer tydligt

Reply:
Varfør gør vi saker som vi kanske inte har behov av? 
Det ær væl troligtvis samma info som skal hæmtas men jag ønskar bara att vi inte skickar fnr i header :)

Reply:
Jepp - vi bør endre på endepunkt i integrasjoner. kan evt. lage en ny oppgave for dette? 
 
Nei - vi burde ikke lage ting vi ikke har bruk for 👍 
Derfor lager vi en liten prototype som vi kan få feedback på fra saksbehandlere og fag (når vi ser bruk). 
Det er mulig dette er mer forvirrende- enn oppklarende for saksbehandlere -> i så fall må vi fjerne, eller forbedre. 

Vi er ganske sikre på at gyldige vedtak ang. gyldig medlemskap er viktig. Derfor har vi tatt med dette i første omgang. 

Forbedre kan bety: legg til data som f.eks under vurdering, eller lovvalg som er endelig (ikke trolig det er relevant for oss, men vi må sjekke med fag/juss). Dette er en start. Siden mye er uklart har vi valgt å skyve på endringer i integrasjon.

Reply:
Har tenkt - jepp - vi burde endre fnr i header. Enig.